[Bug]: Unable to install the module

I was trying to install the module by running the following commands but it's failing:

```bash
pnpm add nuxt-convex
```

I tried following as well

``` bash
npx nuxt@latest module add convex
```

Error
 
ERR_PNPM_SPEC_NOT_SUPPORTED_BY_ANY_RESOLVER  @nuxt/kit@catalog:prod isn't supported by any available resolver.

This error happened while installing the dependencies of nuxt-convex@0.0.4

An external package outside of the pnpm workspace declared a dependency using
the catalog protocol. This is likely a bug in that external package. Only
packages within the pnpm workspace may use catalogs. Usages of the catalog
protocol are replaced with real specifiers on 'pnpm publish'.

This is likely a bug in the publishing automation of this package. Consider filing
a bug with the authors of:

  nuxt-convex@0.0.4

Progress: resolved 31, reused 31, downloaded 0, added 0

I'm on Windows machine

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -46,4 +46,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Publish to npm
-        run: npm publish --provenance --access public
+        run: pnpm publish --no-git-checks --provenance --access public

